src/scrapers/mercantile.ts

```diff
@@ -0,0 +1,13 @@
+import DiscountScraper from './discount';
+
+type ScraperSpecificCredentials = { id: string, password: string, num: string };
+class MercantileScraper extends DiscountScraper {
+  getLoginOptions(credentials: ScraperSpecificCredentials) {
+    return {
+      ...super.getLoginOptions(credentials),
+      loginUrl: 'https://start.telebank.co.il/login/?bank=m',
+    };
+  }
+}
+
+export default MercantileScraper;
```

Comment:
OMG I postponed this review for long time, just to finally review it and see it based on another scraper??

Sorry!!